src/bin/pg_autoctl/cli_show.c

```diff
@@ -555,6 +568,184 @@ cli_show_nodes(int argc, char **argv)
 }
 
 
+/*
+ * cli_show_nodes_getopts parses the command line options for the
+ * command `pg_autoctl show nodes`.
+ */
+static int
+cli_show_standby_names_getopts(int argc, char **argv)
+{
+	KeeperConfig options = { 0 };
+	int c, option_index = 0, errors = 0;
+	int verboseCount = 0;
+
+	static struct option long_options[] = {
+		{ "pgdata", required_argument, NULL, 'D' },
+		{ "formation", required_argument, NULL, 'f' },
+		{ "group", required_argument, NULL, 'g' },
+		{ "version", no_argument, NULL, 'V' },
+		{ "verbose", no_argument, NULL, 'v' },
+		{ "quiet", no_argument, NULL, 'q' },
+		{ "help", no_argument, NULL, 'h' },
+		{ NULL, 0, NULL, 0 }
+	};
+
+	/* set default values for our options, when we have some */
+	options.network_partition_timeout = -1;
```

Comment:
we should probably have a macro to reset these. there are many copy pastes of this block

Reply:
Also we don't have support for JSON in that command, we should probably add that.